openeo/extra/job_management.py

```diff
@@ -189,14 +199,15 @@ def _normalize_df(self, df: pd.DataFrame) -> pd.DataFrame:
         required_with_default = [
             ("status", "not_started"),
             ("id", None),
-            ("start_time", None),
+            ("start_time", None), 
```

Comment:
```suggestion
            ("start_time", None),
```
apparently you added a trailing space here